java/src/main/native/src/TableJni.cpp

```diff
@@ -1299,9 +1299,11 @@ JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_readCSV(
   CATCH_STD(env, NULL);
 }
 
-JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_readParquet(
-    JNIEnv *env, jclass, jobjectArray filter_col_names, jstring inputfilepath, jlong buffer,
-    jlong buffer_length, jint unit, jboolean strict_decimal_types) {
+JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_readParquet(JNIEnv *env, jclass,
+                                                                   jobjectArray filter_col_names,
+                                                                   jstring inputfilepath,
+                                                                   jlong buffer,
+                                                                   jlong buffer_length, jint unit) {
```

Comment:
This is a breaking change in signature. I am surprised that the Java tests are running. I would have expected a corresponding change in the `Table` Java class's native method signature.

Reply:
Good point. When looking at the java side maybe I can take this into consideration. I need to evaluate how much of a change it is on java and plugin side.

Reply:
[Here](https://github.com/rapidsai/cudf/blob/branch-22.02/java/src/main/java/ai/rapids/cudf/Table.java#L247) is the declaration of the native method in the Java `Table` class.

[Here](https://github.com/rapidsai/cudf/blob/branch-22.02/java/src/main/java/ai/rapids/cudf/Table.java#L868) is where that native call is invoked.

The fix might be as simple as to drop the last parameter in the native function declaration, and remove that argument at the invocation site.

Reply:
Absolutely. Just wanted to be sure of invocation sites but rest looks straight forward.

Reply:
Thanks for the pointers. I'll update the call sites. 👍

Reply:
Made the updates. The parameter was used in only one test.

Reply:
> Absolutely. Just wanted to be sure of invocation sites but rest looks straight forward.

:] @kuhushukla: Sorry, you might have found it strange that I was pointing out the Java code to you (of all people). That comment was addressed to @vuule. The confusion is regretted. :D